docs/docs/reference/client-config.rst

```diff
@@ -143,6 +144,49 @@ data-isso-feed
     thread. The link will only be valid if the appropriate setting, in
     ``[rss]`` section, is also enabled server-side.
 
+data-isso-page-author-hashes
+    Provide the hash (or list of hashes) of the current page's author. Any
+    comments made by those authors will be given the ``isso-is-page-author``
+    class. This can be styled using CSS.
+
+    The hash of a user can be found by checking the ``data-hash`` parameter on the
```

Comment:
Well done, very nicely documented!

Note to myself: The specifics of this should probably be moved into the "Tips & Tricks" section. I'll do that afterwards, not part of this PR.

Also, a script to derive the hashes should be added to contrib or as a CLI option that takes `salt` and other set options into account (also by me):
```python
import codecs
from hashlib import pbkdf2_hmac as pbkdf2
val = "email@email.com".encode("utf-8")  # change me!
salt = b"Eech7co8Ohloopo9Ol6baimi"
iterations = 1000
dklen = 6
func = "sha1"
compute = pbkdf2(
    hash_name=func,
    password=val,
    salt=salt,
    iterations=iterations,
    dklen=dklen
)
hash_ = compute
uhash = codecs.encode(hash_, "hex_codec").decode("utf-8")
print("uhash: %s" % uhash)
```

Reply:
Do I need to add that script or will you do it in another commit? And also is there anything else that needs to be modified in this PR?

Reply:
I'll add that script later. Apart from the missing `.isso-page-author-suffix` styles I mentioned inline, I think this should be good to go.

All in all, this PR is a good one - I know that having a proper user management would ofc be the better option, but within the restrictions of the current project status, this is what makes most sense.

Reply:
The `.isso-page-author-suffix` style has been added in 63d2198ea2e7fe1c70fa44f3a240d832b4865abf.